### DISCUSSION TITLE
Motion directions

### DISCUSSION THREAD

**Discussion by nguyentrangia** [blakeblackshear/frigate#10905] [Ideas]
Can we have additional information about motion like directions NEWS?

**Comment by NickM-27** [comment]
this is very dependent on camera perspective and I'm not sure how useful it would be. What are you looking to use that information for?

**Reply by nguyentrangia** [reply]
Sorry for the confusion. I meant the direction of the camera view such as moving left right up down. I have many use cases to count the number of entering and exiting the door after I setup the camera view correctly. Or moving to a specific place in the view.

**Reply by NickM-27** [reply]
you can already do that using the /events topic and mqtt because `entered_zones` is in chronological order

I am not sure that an HA sensor would provide enough context to be useful
